src/pages/Ingame/index.js

```diff
@@ -12,40 +26,27 @@ export default function Game() {
           <span>📒</span>
         </li>
       </Header>
-      <Teams>
-        <Team>
-          <TeamName>1</TeamName>
-          <SelectedCard>
-            <Card className="selected-card">X</Card>
-          </SelectedCard>
-        </Team>
-        <Team>
-          <TeamName>2</TeamName>
-          <SelectedCard>
-            <Card className="enemy-card">?</Card>
-          </SelectedCard>
-        </Team>
-        <Team>
-          <TeamName>3</TeamName>
-          <SelectedCard>
-            <Card className="enemy-card">X</Card>
-          </SelectedCard>
-        </Team>
-        <Team>
-          <TeamName>4</TeamName>
-          <SelectedCard>
-            <Card className="enemy-card">X</Card>
-          </SelectedCard>
-        </Team>
-      </Teams>
+      <GameItems>
+        {[1, 2, 3, 4].map((it) => (
+          <GameItem id={it} key={`team_${it}`} isSubmitted={isSubmitted} mycard={mycard} />
```

Comment:
이 부분도 뭔가 알기 쉽게  props네임을 설정 해주시면 좋을 것 같습니다. 
```suggestion
          <GameItem id={item} key={`team_${item}`} isSubmitted={isSubmitted} mycard={mycard} />
```